### PR TITLE
Add fix for getAbsolutePath() and heavy files download from icoomon

### PR DESCRIPTION
**Pull request by ptimatth**
Fix for issue #15 and fix for heavy files takes from fork of @omarbenn

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -203,9 +206,6 @@ async function pipeline(options = {}) {
       await sleep(1000);
     }
     // reload the page let icomoon read latest indexedDB data
-    await c.send('Page.reload');
```
Comment (Yuyz0112) [inline index.js]: I'm not using icomoon-cli any more, are this operations useless now?

Comment (ptimatth) [reply index.js]: Yap, we have tested and it make more issue than it solve

**Comment by Yuyz0112** [comment]
Thanks

**Comment by ptimatth** [comment]
Yen could you deploy this new version on npmjs.com ?

**Pull request merged**